apps/rpc/src/modules/event/attendance-service.ts

```diff
@@ -137,6 +137,7 @@ export interface AttendanceService {
   updateAttendeeById(handle: DBHandle, attendeeId: AttendeeId, data: Partial<AttendeeWrite>): Promise<Attendee>
   executeReserveAttendeeTask(handle: DBHandle, task: InferTaskData<ReserveAttendeeTaskDefinition>): Promise<void>
   deregisterAttendee(handle: DBHandle, attendeeId: AttendeeId, options: EventDeregistrationOptions): Promise<void>
+  findChargeAttendeeScheduleDate(handle: DBHandle, attendeeId: AttendeeId): Promise<Date | null>
```

Comment:
This should probably be just be stored on attendance instead of having to query the task system.

I also suspect this function to be really slow because we have no indexing on any of the JSON column fields